tests/Bynder/Api/Impl/BynderClientTest.php

```diff
@@ -92,6 +97,11 @@ public function testGetAuthorizationUrl()
      */
     public function testGetAccessTokenClientCredentials()
     {
+        // This should follow client_credentials flow as redirectUri is invalid
+        $requestHandler = $this->setUpRequestHandler('client_credentials', ' ');
+        $this->getAccessToken($requestHandler);
+        
+        // This should follow client_credentials flow as redirectUri is missing
```

Comment:
Shouldn't we have different mock responses for the client credentials and auth code cases?

Now it calls the exact same `this->getAccessToken()` so there's no way we can verify that it indeed followed client_credentials flow in the first case I think.

Reply:
Well the token response is still the same we can have an echo statement to check which is used.

Reply:
To be even more explicit I modified the expected request for the mock by altering `setUpRequestHandler` slightly to check for `code` which is only present under `authorization_flow` This should be sufficient to distinguish between the 2 grant type flows imo

Reply:
I see now :) Think I misread the code a bit yesterday